src/pages/debrief/cat-be/debrief.cat-be.selector.ts

```diff
@@ -84,16 +84,17 @@ export const getVehicleCheckDangerousFaults =
     if (!vehicleChecks) {
```

Comment:
This file is doing a lot of additional logic which should really live in a provider - is there any chance you can add to TODO to look at moving the logic

Reply:
yeah think it can go into the `FaultCountProvider`

Reply:
Added todo